SimpleTimer.cs

```diff
@@ -33,22 +34,30 @@ public void Start()
             }
             
             isRunning = true;
-            onTimerStart?.Invoke(currentTime);
+            Execute(onTimerStart, currentTime);
 
             StaticCoroutineHandler.StartCoroutineStatic(Timer());
         }
 
+        private void Execute(Action<float> action, float value) {
```

Comment:
I'm not sure that it's a good idea to avoid rethrowing exceptions in Start, but it's not super important. Thank you for your changes.